assets/src/scripts/components/graph/state/layout.js

```diff
@@ -12,6 +12,10 @@ const VIEWPORT_RESET = `${MOUNT_POINT}/VIEWPORT_RESET`;
 const VIEWPORT_SET = `${MOUNT_POINT}/VIEWPORT_SET`;
 
 
+const getNamespace = (state) => {
```

Comment:
Consider the more concise form of the arrow function: const getNamespace = state => (state || {})[MOUNT_POINT] || {};

Reply:
sure